src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Lottery and ColorSupplier are different things. Lottery should use ColorSupplier, so, add ColorSupplier field to Lottery class
```suggestion
public class Lottery {
```